unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -4359,22 +4359,31 @@ float allowedError
                 return;
             }
 
-            var metadata = assetDb.prefabs.Where(p => p.GetComponent<SimObjPhysics>() != null).Select(
-                p => {
-                    var simObj = p.GetComponent<SimObjPhysics>();
-                    var bb = simObj.AxisAlignedBoundingBox;
-                    return new AssetMetadata() {
-                        id = simObj.gameObject.name,
-                        type = simObj.Type.ToString(),
-                        primaryProperty = simObj.PrimaryProperty.ToString(),
-                        secondaryProperties = simObj.SecondaryProperties.Select(s => s.ToString()).ToList(),
-                        boundingBox = new BoundingBox() {
-                            min = bb.center - bb.size / 2.0f,
-                            max = bb.center + bb.size / 2.0f
-                        }
-                    };
+            var metadata = new Dictionary<string, Dictionary<string, object>>();
+            foreach (GameObject p in assetDb.prefabs) {
+                if (p.GetComponent<SimObjPhysics>() == null) {
```

Comment:
assetDb prefabs are already filtered by already having SimObjPhysics component

Reply:
Ok, I can remove this check. I was just copying it from: `.Where(p => p.GetComponent<SimObjPhysics>()`
![image](https://user-images.githubusercontent.com/28768645/133344027-d3858c29-f97c-4c51-ba28-624c691a556b.png)

Reply:
Removed in https://github.com/allenai/ai2thor/pull/887/commits/26ebe71678f71162b7bd07be6c612f5c6bf11e18.

Reply:
That makes sense thinking back about it I think the check is good as we are assuming what an asset is, It was just my first thought but probably better to have

Reply:
re-added here: https://github.com/allenai/ai2thor/pull/887/commits/fa1542e374dc14d26bc16affc7c2c5ef258cfef1